src/uct/api/v2/uct_v2.h

```diff
@@ -577,6 +588,40 @@ typedef struct uct_iface_is_reachable_params {
 } uct_iface_is_reachable_params_t;
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Operation parameters passed to @ref uct_ep_is_connected.
+ */
+typedef struct uct_ep_is_connected_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_ep_is_connected_field_mask_t. Fields not specified in this
+     * mask will be ignored. Provides ABI compatibility with respect to adding
+     * new fields.
+     */
+    uint64_t                 field_mask;
+
+    /**
+     * Device address to check for connectivity.
+     * This field must not be passed if iface_attr.dev_addr_len == 0.
```

Comment:
All these `This field...` remarks seem confusing. So if addr_len==0, must not pass anything? Maybe clarify that above?